src/humam/figures/figure_connectivity_matrices.py

```diff
@@ -9,7 +9,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from matplotlib.colors import LogNorm
-from data_loader.dk_fullnames_to_shortnames import dk_full_to_short
+
+from humam.data_loader.dk_fullnames_to_shortnames import dk_full_to_short
```

Comment:
Why different convention here? The other files use `import humam.dataloader as dl` and `dl.…` in the code.